editor/src/components/navigator/navigator-utils.ts

```diff
@@ -256,7 +289,20 @@ export function getNavigatorTargets(
           }
         }
       } else {
-        fastForEach(Object.values(subTree.children), (child) => {
+        const notProcessedChildren = Object.values(subTree.children).filter(
+          (c) => !processedPathsAsRenderProp.has(c.pathString),
+        )
+        if (
+          propertyControls != null &&
+          Object.keys(propertyControls).length > 0 &&
+          notProcessedChildren.length > 0 &&
+          renderPropFound // only show a dedicated label for the children prop if the component has render props too
+        ) {
+          const entry = renderPropNavigatorEntry(EP.appendToPath(path, 'children'), 'children')
```

Comment:
I don't think it's worth fixing now, but I think there is something lopsided in the idea that we decide to show / hide a label here in the navigator entries generation. I'd rather we always have the label and then hide it conditionally. but I guess that would make drag and drop less straightforward

Reply:
adding a label for children is very noisy visually, I think it's better to add a label for the children prop only if there's a render prop right above in the navigator

<img width="277" alt="image" src="https://github.com/concrete-utopia/utopia/assets/16385508/abc8f26a-0aa9-4878-8ed2-ac43bb625c72">

Reply:
> I'd rather we always have the label and then hide it conditionally
so what I meant is to add the label in the navigator entries, and then hide it via css if there is no render prop

Reply:
I tried that too, the reason it doesn't work is that because the `AutoSizer` in the Navigator component needs to know the size of each item beforehand, hiding the child label via css would leave a gap in the place of the child label:

<img width="358" alt="image" src="https://github.com/concrete-utopia/utopia/assets/16385508/4f56a2fe-7a66-46cf-b3dd-be23f996075d">


(of course if you have a specific solution in mind, feel free to share it)

Reply:
nah, I think we are good with the current one, just wanted to write that if _you_ have an idea of how to separate these two concerns, I'd prefer that. but it's good as is